airbyte-webapp/tsconfig.json

```diff
@@ -24,7 +24,8 @@
     "noImplicitReturns": true,
     "noImplicitThis": true,
     "noImplicitAny": true,
-    "noFallthroughCasesInSwitch": true
+    "noFallthroughCasesInSwitch": true,
+    "useUnknownInCatchVariables": false
```

Comment:
:information_source: By default TypeScript will now type the catch variable as `unknown` instead of `any`. This will though require a lot of more code changes whenever we catch exceptions. Thus disabling this for now, and continue typing the catch variable as `any`.